src/components/Resource/ResourceCreate.tsx

```diff
@@ -138,6 +141,15 @@ export default function ResourceCreate(props: resourceProps) {
           }
           return;
         }
+        case "requested_quantity":
+          if (state.form[field]) {
+            const value = state.form[field];
+            if (!value || parseFloat(value) < 1) {
+              errors[field] = `Value Can't be Smaller than 1`;
```

Comment:
Use the `errorText` added above.

Since you are at it, add translations for all the error texts.